requirements.txt

```diff
@@ -4,7 +4,7 @@ matplotlib
 pandas
 bokeh
 # For the site
-devstats==0.1rc3
+devstats @ git+https://github.com/scientific-python/devstats@main
```

Comment:
Why do we have this here if there is a curl command above to actually get the files?

Reply:
The curl command grabs the data, this installs the tool (which contains the report templates).

Maybe we separate the fetching and templates in the future?

Reply:
Ahh, my brain is a mush, of course these are distinct URLs.